public/index.html

```diff
@@ -88,6 +88,52 @@
       <footer>
       </footer>
     </article>
+
+    <article class='female'>
+      <hgroup>
+      <header>💻  “Ruby &amp; The Repository Pattern <em>With Hanami!😊” - Heather Hsin Yi Lim</header>
+      </hgroup>
+
+      <figure>
+        <a title='Heather Lim' href='https://pbs.twimg.com/profile_images/680194548669091841/QMgLf7lg.jpg' rel='modal' target='picture'>
+          <img alt='First Guest' src='https://pbs.twimg.com/profile_images/680194548669091841/QMgLf7lg.jpg'>
+        </a>
+
+        <figcaption>
+          <a title='Heather Lim' rel='twitter' target='twitter' href='http://twitter.com/heatherhsin'>@Heatherhsin</a>
+          Heather Lim
+        </figcaption>
+      </figure>
+
+      <blockquote>
+        <q>Learning Hanami was my personal benefit. If you just know Rails you will always be in that <q>Rails way</q> mindset and that's not necessarily true <strong>All</strong> the time. Choices are <u>always</u> nice!</q>
+      </blockquote>
+
+      <figure>
+        <video controls='yes'></video>
+
+        <figcaption>30:00 minute viewing time</figcaption>
+      </figure>
+
+      <p>Many of us that use Ruby are familiar with using Rails. Ruby is my first language and Rails is my first framework. However, once I started working with another framework in another language, I found that I had been taking ActiveRecord for granted and treating it as the only way to do things. I took this as an opportunity to explore another framework in Ruby.
+
+      <p>Hanami is a lightweight web framework for Ruby. It’s easy to use and productive. Unlike Rails, it doesn’t have models that inherit from ActiveRecord that tend to mix business logic with persistence. Objects are simpler with more clear cut responsibilities. This talk explores creating a simple app with Hanami that shows the repository pattern in action.
+
+      <aside>
```

Comment:
@heatherlim I'll get this cleaned up. This was from an example I used from Jessica Kerr.